mockups/pyGtkMock/guiTry.py

```diff
@@ -0,0 +1,68 @@
+import gi
+gi.require_version('Gtk', '3.0')
+from gi.repository import Gtk, Gdk
+from defcon import Font
+
+
+class FlowBoxWindow(Gtk.Window):
+
+    def __init__(self):
+        Gtk.Window.__init__(self, title="Character Map Demo")
+        self.set_border_width(10)
+        self.set_default_size(800, 800)
+
+        header = Gtk.HeaderBar(title="Character Map")
+        header.set_subtitle("Sample Character Map app")
+        header.props.show_close_button = True
+
+        self.set_titlebar(header)
+
+        scrolled = Gtk.ScrolledWindow()
+        scrolled.set_policy(Gtk.PolicyType.NEVER, Gtk.PolicyType.AUTOMATIC)
+
+        flowbox = Gtk.FlowBox()
```

Comment:
@samdroid-apps <https://github.com/samdroid-apps> what do you think is the
best way to make the grid layout with the currently available features?

On Sun, May 22, 2016 at 5:32 AM Dave Crossland <notifications@github.com>
wrote:

> In mockups/pyGtkMock/guiTry.py
> <https://github.com/sugarlabs/edit-fonts-activity/pull/22#discussion_r64142109>
> :
>
> > +
> > +    def __init__(self):
> > +        Gtk.Window.__init__(self, title="Character Map Demo")
> > +        self.set_border_width(10)
> > +        self.set_default_size(800, 800)
> > +
> > +        header = Gtk.HeaderBar(title="Character Map")
> > +        header.set_subtitle("Sample Character Map app")
> > +        header.props.show_close_button = True
> > +
> > +        self.set_titlebar(header)
> > +
> > +        scrolled = Gtk.ScrolledWindow()
> > +        scrolled.set_policy(Gtk.PolicyType.NEVER, Gtk.PolicyType.AUTOMATIC)
> > +
> > +        flowbox = Gtk.FlowBox()
>
> @samdroid-apps <https://github.com/samdroid-apps> I'm sorry I didn't
> understand you there.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/sugarlabs/edit-fonts-activity/pull/22/files/e9c4ddb38e51587236fbb82b3481b1a91bb18f20#r64142109>
>